### DISCUSSION TITLE
二段階認証が解除できない

### DISCUSSION THREAD

**Discussion by motoso** [increments/qiita-discussions#879] [要望・アイデア・不具合]
https://qiita.com/settings/two_factor_authentication/configure から二段階認証を無効化するボタンを押すと

>サイト/ブログの形式が正しくありません。

と表示され、無効化することができません。
POST /settings/two_factor_authentication/disable が422を返しています

自分のアカウント固有の問題なのか不明なためいったんこちらに報告させていただきました

**Comment by tanaka-asami415** [comment]
@motoso 
お問合せいただき、ありがとうございます。

調査したところ、プロフィールの  `サイト/ブログ` のURL末尾にスペースが入っていたことが原因でした。
昔は許可されていたのですが、現在はスペースは禁止になっているためです。

弊社側でもエラーが出ていたため、motoso 様のQiitaアカウントの  `サイト/ブログ` のURLを修正させていただきました。
もう一度、二段階認証を無効化いただければ解除できるかと思いますので、お手数ですが再度お試しいただけますと幸いです🙇

**Reply by motoso** [reply]
@tanaka-asami415 返信が遅れ申し訳ありません。解除することができました。迅速なご対応、ありがとうございました。
